[CI] NoMasterNodeIT.testNoMasterActionsWriteMasterBlock assertion fails

Build failure: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+intake/2366/console

Reproduce with:

```
./gradlew :server:integTest -Dtests.seed=1120ACA61AAEFC99 -Dtests.class=org.elasticsearch.cluster.NoMasterNodeIT -Dtests.method="testNoMasterActionsWriteMasterBlock" -Dtests.security.manager=true -Dtests.locale=ar-EG -Dtests.timezone=Australia/Melbourne -Dcompiler.java=11 -Druntime.java=8
```

Failure:

```
FAILURE 2.86s J1 | NoMasterNodeIT.testNoMasterActionsWriteMasterBlock <<< FAILURES!
12:03:09    > Throwable #1: java.lang.AssertionError: AcknowledgedResponse failed - not acked
12:03:09    > Expected: <true>
12:03:09    >      but: was <false>
12:03:09    > 	at __randomizedtesting.SeedInfo.seed([1120ACA61AAEFC99:F2165231123B1268]:0)
12:03:09    > 	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
12:03:09    > 	at org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertAcked(ElasticsearchAssertions.java:112)
12:03:09    > 	at org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertAcked(ElasticsearchAssertions.java:116)
12:03:09    > 	at org.elasticsearch.test.TestCluster.wipeIndices(TestCluster.java:142)
12:03:09    > 	at org.elasticsearch.test.TestCluster.wipe(TestCluster.java:79)
12:03:09    > 	at org.elasticsearch.test.ESIntegTestCase.afterInternal(ESIntegTestCase.java:564)
12:03:09    > 	at org.elasticsearch.test.ESIntegTestCase.cleanUpCluster(ESIntegTestCase.java:2165)
12:03:09    > 	at java.lang.Thread.run(Thread.java:748)
```

Could not reproduce locally.

I will follow up with muting the test.

Something very similiar looking happened on 7.1 as well today: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.1+release-tests/3/console but on the "testNoMasterActions"

I couldn't reproduce locally but this is the reproduce line
```
./gradlew :server:integTest --tests "org.elasticsearch.cluster.NoMasterNodeIT.testNoMasterActions" \
  -Dtests.seed=7D0AA22B1B631B12 \
  -Dtests.security.manager=true \
  -Dbuild.snapshot=false \
  -Dtests.jvm.argline="-Dbuild.snapshot=false" \
  -Dtests.locale=no \
  -Dtests.timezone=America/Rainy_River \
  -Dcompiler.java=12 \
  -Druntime.java=8
```
Failure:
```
java.lang.AssertionError: AcknowledgedResponse failed - not acked
Expected: <true>
     but: was <false>
	at __randomizedtesting.SeedInfo.seed([7D0AA22B1B631B12:3938C9E184D237CD]:0)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertAcked(ElasticsearchAssertions.java:112)
	at org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertAcked(ElasticsearchAssertions.java:116)
	at org.elasticsearch.test.TestCluster.wipeIndices(TestCluster.java:142)
	at org.elasticsearch.test.TestCluster.wipe(TestCluster.java:79)
	at org.elasticsearch.test.ESIntegTestCase.afterInternal(ESIntegTestCase.java:594)
	at org.elasticsearch.test.ESIntegTestCase.cleanUpCluster(ESIntegTestCase.java:2196)
```

--- a/server/src/test/java/org/elasticsearch/cluster/NoMasterNodeIT.java
+++ b/server/src/test/java/org/elasticsearch/cluster/NoMasterNodeIT.java
@@ -192,7 +192,6 @@ public class NoMasterNodeIT extends ESIntegTestCase {
         }
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/39688")
     public void testNoMasterActionsWriteMasterBlock() throws Exception {
         Settings settings = Settings.builder()
             .put(AutoCreateIndex.AUTO_CREATE_INDEX_SETTING.getKey(), false)
@@ -264,8 +263,6 @@ public class NoMasterNodeIT extends ESIntegTestCase {
             assertThat(e.status(), equalTo(RestStatus.SERVICE_UNAVAILABLE));
         }
 
-        disruptionScheme.stopDisrupting();
-
-        client().admin().cluster().prepareHealth().setWaitForGreenStatus().setWaitForNodes("3").get();
+        internalCluster().clearDisruptionScheme(true);
     }
 }

